client/src/components/pages/common/Login/index.js

```diff
@@ -1,5 +1,125 @@
 import React from 'react';
+import './index.css';
+import { ThemeProvider } from '@material-ui/core/styles';
+import axios from 'axios';
+import TextField from '@material-ui/core/TextField';
+import { Link } from 'react-router-dom';
+import Theme from './theme';
+import SimpleCard from '../../../common/card';
+import Typography from '../../../common/Typography';
+import Button from '../../../common/Button';
+import loginValidate from '../../../../utils/application/LoginValidation';
 
-const Login = () => <h1>Login</h1>;
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: null,
+      password: null,
+      role: null,
+      message: '',
+    };
+  }
+
+  handleError = () => {
+    window.location.href = '/404';
```

Comment:
delete this row we don't need it anymore

Reply:
as Ahmed's said, there is no need for this function , you can handle error via login function